src/easyscore.ts

```diff
@@ -367,11 +383,12 @@ export class Builder {
     chord.forEach((notePiece: NotePiece, index: number) => {
       const accid = notePiece.accid;
       if (typeof accid === 'string') {
-        const accidental: Accidental = factory.Accidental({ type: accid });
-        // TODO: Remove "as unknown as Modifier".
-        // This compilation warning will be fixed after factory & accidental are migrated to typescript.
-        note.addAccidental(index, accidental as unknown as Modifier);
-        accidentals.push(accidental);
+        // Do not attach sharps or flats they are included in keys.
+        if (sharpFlat(accid) == '') {
```

Comment:
I don't understand what's going on here. Can you add some tests?

Also, can you put this bugfix in a different PR? Let's keep this PR for just the documentation updates.

Reply:
I think this will break EasyScore. Did you check the regression tests?

Reply:
You are right,  I was yesterday so focused on getting the applyAccidentals right that I broke the other ones. I assumed that I was right but there is something missing.

Reply:
removed, I will open an issue to resolve this topic.